<?xml version="1.0" ?>
<!-- Generated using onshape-to-robot -->
<!-- Onshape https://cad.onshape.com/documents/6e82ea959690d515a75aa9d5/w/3a22479ac51efe2ee6ec6cbf/e/449bb5f768d8a2b07e32509f -->
<robot name="bracketbot_v1_rev1">
  <!-- Link base_extrusion -->
  <link name="base_extrusion">
    <inertial>
      <origin xyz="5.0398e-16 0.01055 0.56913" rpy="0 0 0"/>
      <mass value="3.3473"/>
      <inertia ixx="0.73427" ixy="0.000803896" ixz="-2.92746e-16" iyy="0.739173" iyz="3.94633e-17" izz="0.00819634"/>
    </inertial>
    <!-- Part newsimplerhead -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/newsimplerhead.stl"/>
      </geometry>
      <material name="newsimplerhead_material">
        <color rgba="0.917647 0.72549 0.490196 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/newsimplerhead.stl"/>
      </geometry>
    </collision>
    <!-- Part base_extrusion -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/base_extrusion.stl"/>
      </geometry>
      <material name="base_extrusion_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/base_extrusion.stl"/>
      </geometry>
    </collision>
    <!-- Part vert_extrusion_1 -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_1.stl"/>
      </geometry>
      <material name="vert_extrusion_1_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_1.stl"/>
      </geometry>
    </collision>
    <!-- Part vert_extrusion_3 -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_3.stl"/>
      </geometry>
      <material name="vert_extrusion_3_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_3.stl"/>
      </geometry>
    </collision>
    <!-- Part combined_wheelmount -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/combined_wheelmount.stl"/>
      </geometry>
      <material name="combined_wheelmount_material">
        <color rgba="0.917647 0.72549 0.490196 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/combined_wheelmount.stl"/>
      </geometry>
    </collision>
    <!-- Part vert_extrusion_4 -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_4.stl"/>
      </geometry>
      <material name="vert_extrusion_4_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_4.stl"/>
      </geometry>
    </collision>
    <!-- Part part_70 -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/part_70.stl"/>
      </geometry>
      <material name="part_70_material">
        <color rgba="0.917647 0.72549 0.490196 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/part_70.stl"/>
      </geometry>
    </collision>
    <!-- Part vert_extrusion_2 -->
    <visual>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_2.stl"/>
      </geometry>
      <material name="vert_extrusion_2_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="0 0.01055 0.02" rpy="1.98359e-16 1.98523e-23 6.31581e-23"/>
      <geometry>
        <mesh filename="package://assets/vert_extrusion_2.stl"/>
      </geometry>
    </collision>
  </link>
  <!-- Frame depth (dummy link + fixed joint) -->
  <link name="depth">
    <origin xyz="0 0 0" rpy="0 -0 0"/>
    <inertial>
      <origin xyz="0 0 0" rpy="0 0 0"/>
      <mass value="1e-9"/>
      <inertia ixx="0" ixy="0" ixz="0" iyy="0" iyz="0" izz="0"/>
    </inertial>
  </link>
  <joint name="depth_frame" type="fixed">
    <origin xyz="0.033 -0.0208296 1.51466" rpy="-2.19911 9.27872e-17 -3.14159"/>
    <parent link="base_extrusion"/>
    <child link="depth"/>
    <axis xyz="0 0 0"/>
  </joint>
  <!-- Link composite_part_1 -->
  <link name="composite_part_1">
    <inertial>
      <origin xyz="1.48015e-05 2.13538e-05 0.0757492" rpy="0 0 0"/>
      <mass value="3.88752"/>
      <inertia ixx="0.0055964" ixy="-4.65272e-09" ixz="-2.58188e-07" iyy="0.00559573" iyz="4.47551e-06" izz="0.00870403"/>
    </inertial>
    <!-- Part composite_part_1 -->
    <visual>
      <origin xyz="2.89335e-18 0 -0.0023" rpy="1.5708 -0 0"/>
      <geometry>
        <mesh filename="package://assets/composite_part_1.stl"/>
      </geometry>
      <material name="composite_part_1_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="2.89335e-18 0 -0.0023" rpy="1.5708 -0 0"/>
      <geometry>
        <cylinder radius="0.085" length="0.025"/>
      </geometry>
    </collision>
  </link>
  <!-- Joint from base_extrusion to composite_part_1 -->
  <joint name="drive_right" type="continuous">
    <origin xyz="-0.0853 0.01055 0.0468695" rpy="1.5708 -6.27014e-08 -1.5708"/>
    <parent link="base_extrusion"/>
    <child link="composite_part_1"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" velocity="400" lower="-3.141592653589793" upper="3.141592653589793"/>
  </joint>
  <!-- Link composite_part_1_2 -->
  <link name="composite_part_1_2">
    <inertial>
      <origin xyz="1.48015e-05 2.13538e-05 0.0757492" rpy="0 0 0"/>
      <mass value="3.88752"/>
      <inertia ixx="0.0055964" ixy="-4.65272e-09" ixz="-2.58188e-07" iyy="0.00559573" iyz="4.47551e-06" izz="0.00870403"/>
    </inertial>
    <!-- Part composite_part_1_2 -->
    <visual>
      <origin xyz="1.2908e-18 6.93889e-18 -0.0023" rpy="1.5708 -0 0"/>
      <geometry>
        <mesh filename="package://assets/composite_part_1.stl"/>
      </geometry>
      <material name="composite_part_1_2_material">
        <color rgba="0.917647 0.917647 0.917647 1"/>
      </material>
    </visual>
    <collision>
      <origin xyz="1.2908e-18 6.93889e-18 -0.0023" rpy="1.5708 -0 0"/>
      <geometry>
        <cylinder radius="0.085" length="0.025"/>
      </geometry>
    </collision>
  </link>
  <!-- Joint from base_extrusion to composite_part_1_2 -->
  <joint name="drive_left" type="continuous">
    <origin xyz="0.0853 0.01055 0.0468695" rpy="1.5708 9.77997e-17 1.5708"/>
    <parent link="base_extrusion"/>
    <child link="composite_part_1_2"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000" velocity="400" lower="-3.141592653589793" upper="3.141592653589793"/>
  </joint>
</robot>
